src/pages/RFOX/components/Claim/ClaimSelect.tsx

```diff
@@ -0,0 +1,121 @@
+import { Box, Button, CardBody, Flex, Stack } from '@chakra-ui/react'
+import { type AssetId, foxAssetId } from '@shapeshiftoss/caip'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import { TransferType } from '@shapeshiftoss/unchained-client'
+import { type FC, useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIconWithBadge } from 'components/AssetIconWithBadge'
+import { SlideTransition } from 'components/SlideTransition'
+import { RawText } from 'components/Text'
+import { TransactionTypeIcon } from 'components/TransactionHistory/TransactionTypeIcon'
+import { toBaseUnit } from 'lib/math'
+import { selectAssetById, selectFirstAccountIdByChainId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import type { RfoxClaimQuote } from './types'
+import { ClaimRoutePaths, type ClaimRouteProps } from './types'
+
+type ClaimRowProps = {
+  assetId: AssetId
+  amount: string
```

Comment:
```suggestion
  amountCryptoPrecision: string
```

Reply:
Absolutely! Actioned in https://github.com/shapeshift/web/pull/7004/commits/b39ea0eb699d2309beeb8cef365131bd0bba9bab